k8s/aat/common/ccd/api-gateway-web.yaml

```diff
@@ -15,7 +15,7 @@ spec:
   chart:
     repository: https://hmctspublic.azurecr.io/helm/v1/repo/
     name: ccd-api-gateway-web
-    version: 1.0.9
```

Comment:
@hemantt please bump up other microservices accordingly if we are releasing them, for example print service, man web, etc...